src/utils/dispatch.js

```diff
@@ -85,8 +88,22 @@ export const prepareCreateDispatch = (...middlewares) => {
  * https://www.npmjs.com/package/redux-thunk
  */
 export const thunkMiddleware = ({getState, dispatch}) => (next) => action => {
-		if(typeof action === 'function'){
-			return action({state: getState(), dispatch})
-		}
-		return next(action)
+	if(typeof action === 'function'){
+		return action({state: getState(), dispatch})
+	}
+	return next(action)
+}
+
+
+/**
+ * Check if action type contains ERROR, FAILED, FAILURE or WARNING
+ * and send a message to sentry if it does
+ */
+export const sentryMiddleware = ({getState, dispatch}) => (next) => action => {
```

Comment:
Sentry sollte nicht Teil der Middlewares sein, sondern Teil des Error handlings. Es soll nicht im normalen Ablauf getriggert werden. Beim Error handling ist es auch nur ein nice to have. Daher die Error Behandlung wird absolviert und dann wenn alles erledigt ist, wird sentry nachgelagert ausgeführt.

Reply:
Ist dort drin, weil sonst gewisse fehler nicht abgefangen werden, da diese ansonsten nicht auftauchen und abgefangen werden. Das kann zum beispiel das Fehlschlagen einer server abfrage sein, was aber augrund des offline modes auch gar kein fehler werfen darf